t/t1105-proxy.t

```diff
@@ -68,4 +68,11 @@ test_expect_success 'flux-proxy fails with unknown URI path (ENOENT)' '
 	grep "No such file or directory" badpath.err
 '
 
+test_expect_success HAVE_RSH 'flux-proxy forwards LD_LIBRARY_PATH' '
```

Comment:
Could you instead test functionality without `rsh` available by setting something like `FLUX_SSH=/bin/echo`?

Reply:
That's a really interesting idea.  Looks like the output from echo is not passed through:
```
FLUX_SSH=echo ../src/cmd/flux proxy ssh://foo/bar
flux-proxy: ssh://foo/bar: Network dropped connection on reset
```

I could add testing-specific capabilities. Maybe a flag or something that is a functional no-op (no popen) but it echos the command that would run instead.  Thoughts?

Reply:
I'd add a script that logs the arguments to file instead of using echo then:
```
ƒ(s=1,d=0,builddir) grondo@asp:~/git/f.git$ cat proxinator.sh
#!/bin/sh
echo ssh "$@" > proxinator.log
ƒ(s=1,d=0,builddir) grondo@asp:~/git/f.git$ LD_LIBRARY_PATH=blah FLUX_SSH=./proxinator.sh flux proxy ssh://foo/bar
flux-proxy: ssh://foo/bar: Connection reset by peer
ƒ(s=1,d=0,builddir) grondo@asp:~/git/f.git$ cat proxinator.log
ssh foo LD_LIBRARY_PATH=/home/grondo/git/f.git/src/common/.libs:blah /home/grondo/git/f.git/src/cmd/flux relay /bar

```